generator/test_case_generator.go

```diff
@@ -224,6 +229,7 @@ var testTypeToTestConfig = map[string][]testConfig{
 		},
 		{testDir: "./test/fluent", terraformDir: "terraform/eks/daemon/fluent/bit"},
 		{testDir: "./test/fluent", terraformDir: "terraform/eks/daemon/fluent/windows/2022"},
+		{testDir: "./test/fluent", terraformDir: "terraform/eks/daemon/fluent/windows/2025"},
```

Comment:
It might be too early to add these changes since we do not support container insights on Windows 2025.